plugins/common/zpool.sh

```diff
@@ -122,3 +122,8 @@ for zp in $pools; do \
             }
         }'
 done
+
+# pool fragmentation
+zpool list -H -o name,fragmentation | sed 's/\%//g' | while read fraglist; do
```

Comment:
1) I would use the full /sbin/zpool path to be consistent with usage elsewhere in this plugin.
2) this will fail on any version of ZFS that doesn't know about "fragmentation" (which, I think, is delivered with the spacemap_histogram feature, but I'm not positive).  Since this is a common plugin and not OS-specific, we should be careful about this.